modules/logging/src/main/java/io/oasp/module/logging/common/impl/SecureLogging.java

```diff
@@ -0,0 +1,242 @@
+package io.oasp.module.logging.common.impl;
+
+import java.lang.reflect.Method;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.Marker;
+import org.slf4j.MarkerFactory;
+
+/**
+ * Class which provides {@link Marker}s for differential logging. Implements "MultiMarker"s for optimal filtering (if
+ * dependency org.owasp available), or corresponding conventional Markers as a fall back solution.
+ * <P>
+ * Example usage:
+ *
+ * <pre>
+ * {@code
+ * LOG.info(SecureLogging.SECURITY_FAILURE_CONFIDENTIAL, "Confidential Security Failure message.");
+ * }
+ * </pre>
+ *
+ * Example filters for appenders in logback.xml:
+ *
+ * <pre>
+ * {@code
+ * class="org.owasp.security.logging.filter.SecurityMarkerFilter"
+ * class="org.owasp.security.logging.filter.ExcludeClassifiedMarkerFilter"
+ * }
+ * </pre>
+ */
+public class SecureLogging {
```

Comment:
Could you add a simple JUnit Test that at least verifies that logging with such marker actually works at all (classloading, etc. is sane).